### ISSUE TITLE
[Uploadable] multiple files in the same entity

### ISSUE THREAD

**Issue by aitboudad** [doctrine-extensions/DoctrineExtensions#961]
Hey,
Is it possible to upload multiple files in the same entity like:

``` php
    /** 
     * @Gedmo\UploadableFileName
     */
    private $filename;

    /** 
     * @Gedmo\UploadableFileName
     */
    private $thumb;
```

**Comment by FranzBruckner** [comment]
Hi there,
i have implemented this feature rudimentary. All works fine, but there is still some work to let the fork become stable.

The main drawback is broken BC. 
See https://github.com/FranzBruckner/DoctrineExtensions for more details.

**Comment by webdevilopers** [comment]
:+1:

**Comment by tamcy** [comment]
Hi,

I also wrote my own version of multiple uploadables in a single entity. It's different from the one by @FranzBruckner that you don't need any modification to the annotation for single uploadable.
https://github.com/tamcy/DoctrineExtensions
https://github.com/tamcy/DoctrineExtensions/blob/master/doc/uploadable.md

I can create a PR if thing is right, but currently I only added this feature to annotation.

**Comment by l3pp4rd** [comment]
@tamcy if that is backward compatible would surely be glad to merge it. otherwise it could wait till 2.5 release milestone.

**Comment by diegocastro** [comment]
+1 for this.

**Comment by pimolo** [comment]
@l3pp4rd Any reason why this is not merged into v2.4?
Do you want some help to release v2.5?

**Comment by l3pp4rd** [comment]
well, I try to maintain only bug fixes now, I do not work with php anymore, unless someone takes over the maintenance, extensions will remain at 2.4 with minor patches
